solutions/fully-configurable/variables.tf

```diff
@@ -364,6 +364,16 @@ variable "enable_jmx" {
   default     = true
 }
 
+variable "agent_mode" {
+  type        = string
+  description = "The operational mode for the Sysdig agent. Valid values are 'monitor', 'monitor_light', 'troubleshooting', or null. The 'troubleshooting' mode enables additional metrics that are useful for debugging, such as HTTP request metrics (sysdig_container_net_http_url_request_count, sysdig_container_net_http_url_request_time, sysdig_container_net_http_url_error_count). Set to null to use the agent's default mode."
```

Comment:
should not mention "Sysdig". Should be called "Monitoring agent". I don't think you should mention the valid values here, but instead add them as a dropdown box for the DA input in the ibm_catalog.json. I would actually use the new rich text widget (see [here](https://github.com/terraform-ibm-modules/terraform-ibm-base-ocp-vpc/blob/608a4b0f81fd9b048a8c10d57964499a1fbb8e8a/ibm_catalog.json#L770-L779) for an example)

Reply:
Can you also add a "Learn more" link to the docs about this feature? The variable description has too much info in it